README.md

```diff
@@ -100,28 +121,33 @@ f() = sum((sin(i) for i in 1:round(Int, 1000 + 100*randn())))
 ```
 
 ```
-samples: 10000; evals/sample: 1; memory estimate: 0 bytes; allocs estimate: 0
-                         ┌                                        ┐ 
-      [ 8000.0,  9000.0) ┤ 12                                       
-      [ 9000.0, 10000.0) ┤▇ 117                                     
-      [10000.0, 11000.0) ┤▇▇▇▇▇▇▇ 635                               
-      [11000.0, 12000.0) ┤▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 1810                
-      [12000.0, 13000.0) ┤▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 2959   
-      [13000.0, 14000.0) ┤▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 2460         
-   ns [14000.0, 15000.0) ┤▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 1451                    
-      [15000.0, 16000.0) ┤▇▇▇▇▇ 456                                 
-      [16000.0, 17000.0) ┤▇ 89                                      
-      [17000.0, 18000.0) ┤ 9                                        
-      [18000.0, 19000.0) ┤ 1                                        
-      [19000.0, 20000.0) ┤ 0                                        
-      [20000.0, 21000.0) ┤ 1                                        
-                         └                                        ┘ 
-                                          Counts
-min: 8.109 μs (0.00% GC); mean: 12.865 μs (0.00% GC); median: 12.820 μs (0.00% GC); max: 20.459 μs (0.00% GC).
+samples: 10000; evals/sample: 3; memory estimate: 0 bytes; allocs estimate: 0
+ns
+
+ (7030.0  - 7480.0 ]  ▏11
+ (7480.0  - 7930.0 ]  █▍128
+ (7930.0  - 8380.0 ]  ████████▏788
+ (8380.0  - 8830.0 ]  █████████████████████▏2044
+ (8830.0  - 9280.0 ]  ██████████████████████████████ 2916
+ (9280.0  - 9730.0 ]  ███████████████████████▉2309
+ (9730.0  - 10180.0]  ████████████▎1182
+ (10180.0 - 10630.0]  ████▎413
+ (10630.0 - 11080.0]  █▌140
+ (11080.0 - 11530.0]  ▌44
+ (11530.0 - 11980.0]  ▏6
+ (11980.0 - 12430.0]  ▏3
+ (12430.0 - 12880.0]   0
+ (12880.0 - 13330.0]  ▏5
+ (13330.0 - 18330.0]  ▏11
```

Comment:
the outlier change is especially nice here, where on my m1 macbook, I had such severe outliers that @MasonProtter had to generate the plot in order for it to look Gaussian (#6). With the new outlier bin, I can plot it myself 😄